pkgs/games/steam/fhsenv.nix

```diff
@@ -309,6 +309,7 @@ in buildFHSEnv rec {
       description = "Steam dependencies (dummy package, do not use)";
     };
 
+  passthru.steamargs = args;
```

Comment:
that is unrelated to the actual update, isnt it?

Reply:
Yup, only here so I could easily inspect `extraEnv`.

(To be honest, I didn't actually test this yet at all; all I've done is eval this on the mac and do a few remote builds ;) )

Reply:
Did test it now :)

Reply:
Can confirm also from my end, steam picks it up correctly and can be used.